wiki/en/Privacy-Statement.md

```diff
@@ -7,41 +7,43 @@ permalink: "/wiki/Privacy-Statement"
 
 # Privacy Statement
 
-Please note that the English version of this privacy statement is the original and, as such, the binding version. To access the English version, go to the top/top right of this page and click on the "en" link.
-The following only applies to the version you can download from this website, [jamulus.io](https://jamulus.io). 
+**The English version of this document represents the official privacy statement. Should a conflict arise between the English language version and any translation hereof, the English version shall prevail. This statement applies to the _current version_ of the Jamulus software. Earlier versions may have had different privacy terms which are no longer applicable. Users are encouraged to use the most current version of the Jamulus software.**
```

Comment:
I suggest to use simpler English for texts like this.
arise, prevail, applicable,... maybe we find simpler synonyms?